css-typed-om/Overview.bs

```diff
@@ -990,6 +1008,24 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |result|.
 </div>
 
+<div algorithm="CSSNumericValue.type()">
+    The <dfn method for=CSSNumericValue>type()</dfn> method
+    returns a representation of the [=type=] of |this|.
+
+    When called, it must perform the following steps:
+
+    1. Let |result| be a new {{CSSNumericType}} with all attributes set to `null`.
+
+    2. For each |baseType| → |power| in the [=type=] of |this|,
+        1. If |power| is not 0,
+            set |result|'s |baseType| attribute to |power|.
```

Comment:
Not sure if `attribute` is the correct term. Maybe `slot`?

Reply:
The term for dictionaries is "member".  Alternately, since per WebIDL you're allowed to treat dictionaries as ordered maps, you can just say "set |result|[|baseType|] to |power|".